.gitignore

```diff
@@ -24,6 +24,9 @@ _testmain.go
 *.test
 *.prof
 
+#don't ignore exe under test/testdata
```

Comment:
I don't think this is a good idea. The ignore file gets more and more complicated. Really. there is no need to exclude "*.exe" from the start. The only executable ever build dynamically is in the _out_ directory which is already excluded. 

So instead of making things more complicated, start making things simpler and remove the '*.exe' exclusion. 

Mind you, we inherited the ignore file from Minikube and never really took the time to go through it. There are several entries in which could be removed.

Reply:
Sure, i was also thinking the same and could not find any '.exe' files elsewhere, but wanted to keeo things the way they were.